backend/src/resources/resources.ts

```diff
@@ -25,3 +25,22 @@ export const getResourcesFromCourse = async (courseId: string) => {
   const resources = await query.findAll();
   return resources;
 };
+
+export const getResourcesFromCourseAndDifficulty = async (
```

Comment:
nit: This function name threw me off a little bit, maybe `getResourcesFromCourseWithDifficulty`